warp/src/js_exports/raygun.rs

```diff
@@ -744,8 +744,12 @@ impl Message {
         self.inner.lines()
     }
 
-    pub fn attachments(&self) -> JsValue {
-        serde_wasm_bindgen::to_value(&self.inner.attachments()).unwrap()
+    pub fn attachments(&self) -> Vec<JsValue> {
+        self.inner
+            .attachments()
+            .iter()
+            .map(|v| serde_wasm_bindgen::to_value(v).unwrap())
```

Comment:
Since a PR is already up to reduce unwrapping, where possible, we should follow the same scheme here too.

```suggestion
            .filter_map(|v| serde_wasm_bindgen::to_value(v).ok())
```